src/frontend/Felles/Visittkort/Visittkort.tsx

```diff
@@ -93,8 +93,8 @@ const VisittkortComponent: FC<{ data: IPersonopplysninger; behandling?: Behandli
                 data: { personIdent: personIdent },
             }).then((respons: RessursSuksess<ISøkPerson> | RessursFeilet) => {
                 if (respons.status === RessursStatus.SUKSESS) {
-                    if (respons.data?.fagsaker?.length) {
-                        settFagsakId(respons.data.fagsaker[0].fagsakId);
+                    if (respons.data?.fagsakPersonId && respons.data?.fagsaker?.length) {
```

Comment:
Trenger kanskje ikke å sjekke fagsaker her?

```suggestion
                    if (respons.data?.fagsakPersonId) {
```

Reply:
Godt poeng!